plugins/system/redirect/redirect.php

```
@@ -285,6 +285,18 @@ private static function doErrorHandling($error)
 				// In case the url contains double // lets remove it
 				$destination = str_replace(JUri::root() . '/', JUri::root(), $dest);
 
+				// Always count redirect hits
+				$redirect->hits++;
+
+				try
+				{
+					$db->updateObject('#__redirect_links', $redirect, 'id');
+				}
+				catch (Exception $e)
+				{
+					JErrorPage::render(new Exception(JText::_('PLG_SYSTEM_REDIRECT_ERROR_UPDATING_DATABASE'), 500, $e));
+				}
```

```suggestion
				{
				}
```

---

If this is removed, how else will you know there is a a fault in the saving mechanism? This would only occur if the database is corrupt or gone away.

---

"I" as the site owner does not know at all, because the user gets the error, which is not necessary when the redirect works anyways.

---

I have to agree with @bembelimen here, we might could write something to a log so the site owner can check the thing themself?. For now can you please update the suggestion with a comment and not just an empty catch method?